### ISSUE TITLE
Ombi Plex Configuration unable to connect

### ISSUE THREAD

**Issue by zetneteork** [tidusjar/Ombi#2917]
**Describe the bug**
Unable to add Plex media server into Ombi, error with connection. Plex Authorization Token is valid.

**To Reproduce**
1. Go to 'media server/Plex'
2. Configure my internal Plex server (_tested with internal and external IP_)
3. Test connectivity
4. Error `We could not connect to the Plex server xyz!`

**Expected behavior**

**Screenshots**
![2019-04-09 14_47_46-127 0 0 1_3388 - Remote Desktop Connection](https://user-images.githubusercontent.com/6087244/55801930-7477fd80-5ad7-11e9-847a-5b31b5854045.png)

**Logs (Logs directory where Ombi is located)**
```
fail: Ombi.Controllers.External.TesterController[1000]
      Could not test Plex
System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.IO.IOException: The server returned an invalid or unrecognized response.
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Ombi.Api.OmbiHttpClient.SendAsync(HttpRequestMessage request) in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 61
   at Ombi.Api.Api.Request[T](Request request) in C:\projects\requestplex\src\Ombi.Api\Api.cs:line 38
   at Ombi.Api.Plex.PlexApi.GetStatus(String authToken, String uri) in C:\projects\requestplex\src\Ombi.Api.Plex\PlexApi.cs:line 97
   at Ombi.Controllers.External.TesterController.Plex(PlexServers settings) in C:\projects\requestplex\src\Ombi\Controllers\External\TesterController.cs:line 246
```

**Desktop (please complete the following information):**
 - Debian wit Docker [18.09.4]
 - Docker image [v3.0.4256-pkg-d8080b67-ls14](https://github.com/linuxserver/docker-ombi/releases/tag/v3.0.4256-pkg-d8080b67-ls14)

**Ombi Version (please complete the following information):**
 - Version [3.0.4256]
 - Plex [1.15.3.876]

**Comment by tidusjar** [comment]
Check your language set: https://github.com/tidusjar/Ombi/wiki/Common-Errors#proxmox-lxc-containers

**Comment by zetneteork** [comment]
Custom docker container build with extended LANG settings:
**ENV**
```
LC_ALL=en_US.UTF-8
LANG=en_US.UTF-8
LANGUAGE=en_US.UTF-8
```
**/etc/default/locale**
```
LANG="en_US.UTF-8"
LANGUAGE="en_US:en"
```
That gives same output error.
```
fail: Ombi.Controllers.External.TesterController[1000]
      Could not test Plex
System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.IO.IOException: The server returned an invalid or unrecognized response.
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.ReadNextResponseHeaderLineAsync(Boolean foldedHeadersAllowed)
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.SendAsyncCore(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithNtConnectionAuthAsync(HttpConnection connection, HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Ombi.Api.OmbiHttpClient.SendAsync(HttpRequestMessage request) in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 61
   at Ombi.Api.Api.Request[T](Request request) in C:\projects\requestplex\src\Ombi.Api\Api.cs:line 38
   at Ombi.Api.Plex.PlexApi.GetStatus(String authToken, String uri) in C:\projects\requestplex\src\Ombi.Api.Plex\PlexApi.cs:line 97
   at Ombi.Controllers.External.TesterController.Plex(PlexServers settings) in C:\projects\requestplex\src\Ombi\Controllers\External\TesterController.cs:line 246
```

Based on that test it doesn't seem to be related to lang.

**Comment by tidusjar** [comment]
I don't know much about docker, but you need to make sure that Ombi is able to communicate with that IP address and that resolves to your plex instance

**Comment by zetneteork** [comment]
There is direct communication opened for that. That is not a case.

**Comment by tidusjar** [comment]
Tried telling ombi to ignore any SSL errors? Under Ombi settings, also tried enabling the SSL checkbox?

**Comment by tidusjar** [comment]
I am struggling to help with this issue. It's a problem with the connection between Plex and ombi and the response from the Plex server you are getting is not very detailed. This does not seem to be a bug, but more of a configuration issue. How to resolve this, I don't know because it would be local to your setup.

**Comment by TMaddox** [comment]
I had the same issue, that's how I solved it:

1) I figured out that I can´t reach 10.0.0.100:32400/web in my browser
2) I figured out that someone at Plex forgot to redirect http traffic to https (I can reach https://10.0.0.100:32400/web, but not http://10.0.0.100:32400/web, when I try to connect via https I get a security alert, bad certificate (certificate is only valid for: *.[verylongrandomstring mabymytokensoIreplaceditwithuselesstext].plex.direct))
3) If I turn the secure connection setting in plex to either OFF or NOT MANDATORY, I can connect ombi to plex without SSL. If secure connection is ENABLED, I can´t connect (of course not if I turn SSL off in ombi), but neither when I turn it on.
4) Disabled certificate checking in Settings > Ombi
5) I won, damn bugs!
